ANMN/burst_averaged_product/burst_average.py

```diff
@@ -259,23 +259,23 @@ def create_burst_average_netcdf(input_netcdf_file_path, output_dir):
     # set up dimensions and variables
     output_netcdf_obj.createDimension("TIME", len(time_burst_vals))
     var_time = output_netcdf_obj.createVariable("TIME", input_netcdf_obj["TIME"].dtype,
-                                                ("TIME",), fill_value=get_imos_parameter_info('TIME', '_FillValue'))
+                                                ("TIME",))
 
     dimensionless_var = list_dimensionless_var(input_netcdf_obj)
+    # No FillValue for dimensions as for IMOS conventions
     for var in dimensionless_var:
-        output_netcdf_obj.createVariable(var, input_netcdf_obj[var].dtype, ())
+        output_netcdf_obj.createVariable(var, input_netcdf_obj[var].dtype)
         output_netcdf_obj[var][:] = input_netcdf_obj[var][:]
 
     for var in burst_vars.keys():
-        fillvalue = get_imos_parameter_info(var, '_FillValue')
-        if fillvalue == []:
-            fillvalue = 999999.0
+        var_dtype = input_netcdf_obj[var].dtype
+        fillvalue = getattr(input_netcdf_obj[var], '_FillValue', default_fillvals[np.dtype(var_dtype).str[1:]])
```

Comment:
You can just use `None` as the default value. Calling `createVariable` with `fillvalue=None` has the same effect as what you're doing here.